modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -8041,6 +8038,46 @@ public void testPostCustomObjectEntryWithAutoIncrementObjectField()
 		_objectDefinitionLocalService.deleteObjectDefinition(objectDefinition);
 	}
 
+	@Test
+	public void testPostCustomObjectEntryWithDateTimeObjectField()
+		throws Exception {
+
+		User user = GuestOrUserUtil.getGuestOrUser();
+
+		String timeZoneId = user.getTimeZoneId();
+
+		try {
+			user.setTimeZoneId("America/Anchorage"); //GMT-8
```

Comment:
According to https://www.travelmath.com/time-zone/America/Anchorage, the timezone changes depending on the summer time / winter time, so I think your test could fail in the future